### PR TITLE
Update Chinese Localization

### PR DESCRIPTION
**Pull request by auroursa**
This PR includes the latest translations for zh-CN and zh-TW, and cleans up outdated strings.
cc @KuwaLee @cirx1e

### CODE REVIEW

File: src/locale/locales/zh-TW/messages.po [hunk 1]
```diff
@@ -457,14 +477,23 @@ msgstr "一封電子郵件已發送至 {0}。請查閱郵件並在下方輸入
...
 msgstr "發生錯誤"
 
 #: src/components/StarterPack/ProfileStarterPacks.tsx:315
 msgid "An error occurred while generating your starter pack. Want to try again?"
 msgstr "建立您的入門包時發生錯誤。是否要重試？"
 
+#: src/view/com/util/post-embeds/VideoEmbed.tsx:69
+#: src/view/com/util/post-embeds/VideoEmbed.web.tsx:150
+msgid "An error occurred while loading the video. Please try again later."
+msgstr "載入影片時發生錯誤。請稍後重試。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:495]:
```suggestion
msgstr "載入影片時發生錯誤。請稍後再試。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 2]
```diff
@@ -474,35 +503,43 @@ msgstr "儲存 QR Code 時發生錯誤！"
 msgid "An error occurred while trying to follow all"
 msgstr "跟隨所有帳號時發生錯誤"
 
+#: src/state/queries/video/video.ts:112
+msgid "An error occurred while uploading the video."
+msgstr "上傳影片時出現問題。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:508]:
```suggestion
msgstr "上傳影片時發生錯誤。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 3]
```diff
@@ -1689,6 +1737,10 @@ msgstr "顯示名稱"
 msgid "DNS Panel"
 msgstr "DNS 控制台"
 
+#: src/components/dialogs/MutedWords.tsx:302
+msgid "Do not apply this mute word to users you follow"
+msgstr "不對已跟隨的用戶使用此靜音詞彙"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:1742]:
```suggestion
msgstr "不要對已跟隨的用戶使用此靜音詞彙"
```

File: src/locale/locales/zh-TW/messages.po [hunk 4]
```diff
@@ -2094,7 +2157,15 @@ msgstr "展開或摺疊您正在回覆的完整貼文"
 
 #: src/view/screens/NotificationsSettings.tsx:83
 msgid "Experimental: When this preference is enabled, you'll only receive reply and quote notifications from users you follow. We'll continue to add more controls here over time."
-msgstr ""
+msgstr "實驗性選項：啟用此偏好設定後，您將僅收到您追蹤用戶的回覆和引用通知。隨著時間的推移，我們將繼續在此處添加更多控制選項。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2160]:
```suggestion
msgstr "實驗性選項：啟用此偏好設定後，您將僅收到來自您已跟隨用戶的回覆和引用通知。我們將陸續在此新增更多控制選項。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 5]
```diff
@@ -2188,13 +2259,13 @@ msgstr "無法儲存圖片：{0}"
 
 #: src/state/queries/notifications/settings.ts:39
 msgid "Failed to save notification preferences, please try again"
-msgstr ""
+msgstr "無法儲存通知偏好設定，請重試"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2262]:
```suggestion
msgstr "無法儲存通知偏好設定，請再試一次"
```

File: src/locale/locales/zh-TW/messages.po [hunk 6]
```diff
@@ -2949,21 +3019,26 @@ msgstr "語言"
...
 msgstr "最新"
 
-#: src/components/moderation/ScreenHider.tsx:136
+#: src/components/moderation/ScreenHider.tsx:146
 msgid "Learn More"
 msgstr "瞭解詳情"
 
+#: src/view/com/auth/SplashScreen.web.tsx:152
+msgid "Learn more about Bluesky"
+msgstr "了解有關 BlueSky 的更多資訊"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:3028]:
```suggestion
msgstr "了解有關 Bluesky 的更多資訊"
```

File: src/locale/locales/zh-TW/messages.po [hunk 7]
```diff
@@ -3367,30 +3454,42 @@ msgstr "將所有 {displayTag} 貼文靜音"
...
 msgstr "在貼文內容和話題標籤中隱藏該文字"
 
-#: src/components/dialogs/MutedWords.tsx:141
+#: src/components/dialogs/MutedWords.tsx:274
 msgid "Mute this word in tags only"
 msgstr "僅在話題標籤中隱藏該文字"
 
+#: src/components/dialogs/MutedWords.tsx:170
+msgid "Mute this word until you unmute it"
+msgstr "將這個文字靜音直到您解除它"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:3491]:
```suggestion
msgstr "將這個文字靜音，直到您取消靜音為止"
```

File: src/locale/locales/zh-TW/messages.po [hunk 8]
```diff
@@ -4317,9 +4440,9 @@ msgstr "貼文"
 msgid "Posts"
 msgstr "貼文"
 
-#: src/components/dialogs/MutedWords.tsx:89
-msgid "Posts can be muted based on their text, their tags, or both."
-msgstr "可以靜音貼文所包含的文字和標籤。"
+#: src/components/dialogs/MutedWords.tsx:115
+msgid "Posts can be muted based on their text, their tags, or both. We recommend avoiding common words that appear in many posts, since it can result in no posts being shown."
+msgstr "可以根據文字、標籤或結合兩者來靜音特定貼文。我們建議避免添加常用文字，因為這可能會導致你的動態不顯示任何貼文。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:4445]:
```suggestion
msgstr "可以根據文字、標籤或結合兩者來靜音貼文。我們建議避免新增常見的文字，否則可能導致動態不顯示任何貼文。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 9]
```diff
@@ -4465,11 +4588,23 @@ msgstr "比率"
 msgid "Reactivate your account"
 msgstr "重新啟用您的帳號"
 
+#: src/view/com/auth/SplashScreen.web.tsx:157
+msgid "Read the Bluesky blog"
+msgstr "閱讀 BlueSky 部落格"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:4593]:
```suggestion
msgstr "閱讀 Bluesky 部落格"
```

File: src/locale/locales/zh-TW/messages.po [hunk 10]
```diff
@@ -4465,11 +4588,23 @@ msgstr "比率"
...
+
+#: src/screens/Signup/StepInfo/Policies.tsx:59
+msgid "Read the Bluesky Privacy Policy"
+msgstr "閱讀 BlueSky 隱私權政策"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:4597]:
```suggestion
msgstr "閱讀 Bluesky 隱私權政策"
```

File: src/locale/locales/zh-TW/messages.po [hunk 11]
```diff
@@ -4465,11 +4588,23 @@ msgstr "比率"
...
+
+#: src/screens/Signup/StepInfo/Policies.tsx:49
+msgid "Read the Bluesky Terms of Service"
+msgstr "閱讀 BlueSky 服務條款"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:4601]:
```suggestion
msgstr "閱讀 Bluesky 服務條款"
```

File: src/locale/locales/zh-TW/messages.po [hunk 12]
```diff
@@ -4518,28 +4654,32 @@ msgid "Remove embed"
...
-#: src/components/FeedCard.tsx:304
-#: src/view/com/feeds/FeedSourceCard.tsx:312
+#: src/components/FeedCard.tsx:308
+#: src/view/com/feeds/FeedSourceCard.tsx:311
 msgid "Remove from my feeds?"
 msgstr "從我的動態源中刪除？"
 
+#: src/view/com/util/AccountDropdownBtn.tsx:53
+msgid "Remove from quick access?"
+msgstr "從快速存取中移除？"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:4681]:
```suggestion
msgstr "從快速存取中刪除？"
```

File: src/locale/locales/zh-TW/messages.po [hunk 13]
```diff
@@ -5008,23 +5142,27 @@ msgstr "搜尋 Tenor"
...
 
-#: src/components/TagMenu/index.tsx:187
+#: src/components/TagMenu/index.tsx:198
 msgid "See <0>{displayTag}</0> posts by this user"
 msgstr "查看該用戶包含 <0>{displayTag}</0> 的貼文"
 
-#: src/view/screens/SavedFeeds.tsx:187
+#: src/view/com/auth/SplashScreen.web.tsx:162
+msgid "See jobs at Bluesky"
+msgstr "查看 BlueSky 的職缺"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5163]:
```suggestion
msgstr "查看 Bluesky 的職缺"
```

File: src/locale/locales/zh-TW/messages.po [hunk 14]
```diff
@@ -5060,7 +5198,11 @@ msgstr "選擇 GIF"
 msgid "Select GIF \"{0}\""
 msgstr "選擇 GIF「{0}」"
 
-#: src/view/screens/LanguageSettings.tsx:301
+#: src/components/dialogs/MutedWords.tsx:142
+msgid "Select how long to mute this word for."
+msgstr "選取此文字的靜音時間。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5203]:
```suggestion
msgstr "選擇靜音此文字的時間長度。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 15]
```diff
@@ -5088,7 +5230,11 @@ msgstr "選擇用來託管您的資料的服務商。"
 msgid "Select video"
 msgstr "選擇影片"
 
-#: src/view/screens/LanguageSettings.tsx:283
+#: src/components/dialogs/MutedWords.tsx:242
+msgid "Select what content this mute word should apply to."
+msgstr "選取此靜音文字應用於哪些內容。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5235]:
```suggestion
msgstr "選擇此靜音文字應套用於哪些內容。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 16]
```diff
@@ -5804,6 +5946,10 @@ msgstr "找不到那個入門包。"
 msgid "The account will be able to interact with you after unblocking."
 msgstr "解除封鎖後，該帳號將能夠與您互動。"
 
+#: src/screens/Moderation/index.tsx:368
+msgid "The Bluesky web application"
+msgstr "BlueSky 網頁應用程式"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5951]:
```suggestion
msgstr "Bluesky 網頁應用程式"
```

File: src/locale/locales/zh-TW/messages.po [hunk 17]
```diff
@@ -6123,16 +6277,20 @@ msgstr "這是新來的用戶，請按此瞭解更多有關他們何時加入的
...
-#: src/components/dialogs/MutedWords.tsx:283
-msgid "This will delete {0} from your muted words. You can always add it back later."
-msgstr "這將從您的靜音文字中刪除 {0}，您隨時可以新增回來。"
+#: src/components/dialogs/MutedWords.tsx:435
+msgid "This will delete \"{0}\" from your muted words. You can always add it back later."
+msgstr "這將從您的靜音文字中刪除 \"{0}\"，您隨時可以新增回來。"
+
+#: src/view/com/util/AccountDropdownBtn.tsx:55
+msgid "This will remove @{0} from the quick access list."
+msgstr "這將會從快速存取清單中移除 @{0}。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:6286]:
```suggestion
msgstr "這將從快速存取清單中刪除 @{0}。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 18]
```diff
@@ -7098,6 +7275,10 @@ msgstr "您可以將您的帳號儲存庫下載為一個「CAR」檔案。該檔
 msgid "Your birth date"
 msgstr "您的生日"
 
+#: src/view/com/util/post-embeds/VideoEmbed.web.tsx:145
+msgid "Your browser does not support the video format. Please try a different browser."
+msgstr "您的瀏覽器不支援該影片格式。請嘗試其他瀏覽器。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:7280]:
```suggestion
msgstr "您的瀏覽器不支援該影片格式。請嘗試使用其他瀏覽器。"
```

**Comment by cirx1e** [comment]
please take a look at my review @KuwaLee

**Comment by auroursa** [comment]
@cirx1e Apologies for my delayed review. Since the maintainer of the Traditional Chinese translation hasn't had much time recently, I'll be merging your review this time.
Although I'm not from Taiwan, given your continuous help, I'll accept all of your translation suggestions, thank you!

**Comment by auroursa** [comment]
I believe we are ready to go.

**Comment by KuwaLee** [comment]
@cirx1e 

Sorry, I'm busy on preparing for college entrance affair, I can't review and update timely. We temporarily accept all you change and pull request if you wanna contribute and update this translation project until I can go back.

**Review by pfrazee — APPROVED** [review]
Going to go ahead and pull this in for the next release. Thanks as ever yall

**Pull request merged**